src/content/components/FeatureView/FeatureView.js

```diff
@@ -394,14 +400,18 @@ export class FeatureView extends React.PureComponent {
     return result;
   }
 
-  async getBugs(id) {
+  getBugs(id) {
```

Comment:
I converted the awaits to callbacks since we don't want to wait for the cache get before starting the fetch (which is reliably much slower). So it doesn't need to be async anymore, and async functions are slightly slower due to the promise wrapping. But for the sake of consistency it might be worth keeping it async, since some of these bug view query methods do need to be async